### DISCUSSION TITLE
Player with composition with <Video> only plays the first second then stops

### DISCUSSION THREAD

**Discussion by JoaoTareco** [remotion-dev/remotion#3484] [Q&A]
I have this composition:
```
export const Main = ({ title }: z.infer<typeof CompositionProps>) => {

  const titleStyle: React.CSSProperties = useMemo(() => {
    return { fontFamily, fontSize: 70 };
  }, []);

  return (
    <AbsoluteFill style={container}>
        <TextFade>
          <h1 style={titleStyle}>{title}</h1>
        </TextFade>
           <Video preload="true" src="https://X.supabase.co/storage/v1/object/public/content-bank/user_2/IMG_2200.mp4" />
    </AbsoluteFill>
  );
};
```

Which I am trying to preview using the Player in a Next.js app.

However, when the video starts, the background video stops after the first second, but the text continues to render. Also, if I stop the player, the frame of the background video jumps to whatever it should be on.

I have also tried without preload and OffthreadVideo, but nothing seems to work.

I am accessing my next.js app in chrome.

How could I fix this Player preview issue?

**Comment by JonnyBurger** [comment]
Hi Joao!

I think you suffer from a bug we had in the 4.0.111 release.
It's fixed in 4.0.112. Run `npx remotion upgrade` to get the new release.

Let me know if this doesn't help!

**Reply by JoaoTareco** [reply]
Hello! Thank you for answering!

 Didn't seem to fix it 😕 The issue persists

**Reply by JonnyBurger** [reply]
@JoaoTareco Hello, just making sure (v4.0.112 was just released 10 minutes ago, after I wrote the comment)

Do you have the latest when running `npx remotion versions`, and can you also try clearing the cache once? `rm -rf .next` If you say the issue persists, I investigate

**Reply by JoaoTareco** [reply]
@JonnyBurger it still persists 😕
<img width="470" alt="image" src="https://github.com/remotion-dev/remotion/assets/148398041/ff67ad02-0aa6-49c5-852f-1bd4188488f5">

Here is what is happening:

https://github.com/remotion-dev/remotion/assets/148398041/a5a73567-9082-4202-8060-6b290118eea0

**Reply by JonnyBurger** [reply]
@JoaoTareco You are right. Sorry about this. The fix was not actually merged. Will release 4.0.113 with the bug. Watch out on remotion.dev/changelog

**Reply by JoaoTareco** [reply]
No worries @JonnyBurger, thank you for the great support :)! I will keep an eye on the changelog

**Reply by JoaoTareco** [reply]
@JonnyBurger I've tested with 4.0.113, but the error still persists. Maybe what I am facing is a different issue? It is still as I previously described

**Reply by JonnyBurger** [reply]
@JoaoTareco 🫣 Sorry to make you go through this!
You are right, and only 4.0.114 now contains the actual fix.

**Reply by JoaoTareco** [reply]
Now its working @JonnyBurger , thank you for the support!!
